static/js/controllers/inbox.js

```diff
@@ -467,12 +467,9 @@ var feedback = require('../modules/feedback'),
           controller: 'DeleteDocConfirm',
           model: { docs: docs }
         }).then(function() {
-          if ($state.includes('contacts') || $state.includes('reports')) {
-            if ($scope.selectMode) {
-              $scope.clearSelected();
-            } else {
-              $state.go($state.current.name, { id: null });
-            }
+          if (!$scope.selectMode &&
```

Comment:
Don't clear selected when an item is deleted in selectMode because multiple records might be selected and this stuffs up the subsequent changes. Let the individual handlers clean up after themselves.